src/app/common/schemas.py

```diff
@@ -31,18 +41,35 @@ class PNGRequest(BaseModel):
     @field_validator("filename", mode="before")
     @classmethod
     def validate_filename(cls, value: str) -> str:
+        """Ensure the filename is not empty."""
         return validate_filename(value)
 
     class Config:
         """Make to allow using non-standard types like Path."""
 
-        arbitrary_types_allowed = True  # This allows using non-standard types like Path
+        arbitrary_types_allowed: bool = True  # This allows using non-standard types like Path
 
 
 class PNGResponse(BaseModel):
-    """Response schema for the create_pngs endpoint."""
+    """Response schema for the `create_pngs` endpoint, representing the output of PNG file creation and storage.
 
-    keys: list[str]  # keys in the S3 bucket
+    This schema lists the keys (identifiers) of the created PNG files stored in an S3 bucket,
+    enabling users to retrieve or reference them as needed.
+    """
+
+    keys: list[str] = Field(
+        ...,
+        description="""
+            List of unique identifiers (keys) for the generated PNG files stored in the S3 bucket. Each key allows 
+            access to a specific file within the bucket.
+        """,
+        example=[
+            "dataextraction/file1-1.png",
+            "dataextraction/file1-2.png",
+            "dataextraction/file2-1.png",
+            "dataextraction/file3-1.png",
```

Comment:
do we ever have more than one file in a single request/response?

Reply:
I got carried ways here indeed. Very good comment. Thank you.